### PR TITLE
Add US_ID and US_TN to staging spotlight

### PR DESCRIPTION
**Pull request by colincadams**
## Description of the change

This basically just copies the source for North Dakota to make one for Idaho and Tennessee. I did remove some FTR specific graphs, and pull the facility and supervision districts that are currently in the views (although the lists are long :D)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes [#XXXX]

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Review by lilidworkin — COMMENTED** [review]
Would it be better to factor out some of this contentApi/sources into shared files, rather than copying/pasting for each tenant? I imagine this might make it really hard to update the copy for a page or viz that is shared among all tenants, because you have to make the same change in all five files now, right?

Not suggesting we do this refactor in this PR, but just wondering if there's a good reason not to do this.

File: .github/workflows/spotlight-staging.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
           REACT_APP_AUTH_ENABLED: true
           REACT_APP_AUTH_ENV: development
           REACT_APP_API_URL: ${{ secrets.REACT_APP_API_URL }}
-          REACT_APP_ENABLED_TENANTS: US_ND,US_PA
+          REACT_APP_ENABLED_TENANTS: US_ID,US_ND,US_PA,US_TN
```
Comment (macfarlandian) [inline .github/workflows/spotlight-staging.yml:33]: in addition to this you should update the config files in the vault for staging and local development

**Review by macfarlandian — APPROVED** [review]
this looks ... surprisingly decent! TN data is kinda busted but still in better shape than PA data was at this stage

**Comment by macfarlandian** [comment]
to @lilidworkin 's point, we don't necessarily expect copy to be exactly the same from state to state; we weren't really sure how formulaic it would be in practice, given the variability of state input, so we started off with the copy-and-paste strategy for maximum customizability. With two states it wasn't really worth the effort to come up with a templating system that would work, but if a third or fourth state tips the balance, there is no special reason not to do it? I would just worry about making it overly rigid given that the copy is subject to external approval and this is probably not something we are going to really hold the line on. (The methodology configuration is an example of where I tried to factor out true boilerplate versus bits that could conceivably be state-specific ... it's not pretty IMO)

**Comment by hobuobi** [comment]
@colincadams and I reviewed earlier today and agree that this should be pretty much good to go! 
The only thing that I would like to see included in this PR would be getting the recidivism/reincarceration charts working; Colin is investigating and if feasible to include with a small amount of effort, we should include it. 
All other issues with existing charts should be resolved by a data analyst.

File: spotlight-client/src/contentApi/sources/us_id.ts [hunk 2]
```diff
@@ -0,0 +1,796 @@
...
+the “controlling” charge, meaning the sentence associated with this charge is currently the
+most restrictive sentence being served. For individuals currently incarcerated, the sentence
+associated with the controlling charge determines the judicial district in which the person is
+counted. Individuals on supervision are counted in each judicial district for which they
+are serving a sentence.</p>`;
+
+const prisonBoilerplate = `<p>This data concerns individuals in prison in Idaho.
+It does not include individuals incarcerated in county jails.</p>`;
+
+const probationBoilerplate = `<p>This data may include some individuals on probation in North
```
Comment (terryttsai) [inline spotlight-client/src/contentApi/sources/us_id.ts:41]: Should North Dakota be replaced with Idaho?

File: spotlight-client/src/contentApi/sources/us_id.ts [hunk 3]
```diff
@@ -0,0 +1,796 @@
...
+Dakota as part of the interstate compact program, in which Idaho agrees to supervise
+individuals who were charged and sentenced in a different state, but have a compelling reason
+to serve their probation in Idaho. Under the same program, individuals placed on probation
+from a Idaho court may occasionally serve their probation in a different state.</p>`;
+
+const probationDistrictBoilerplate = `<p>The districts associated with this data indicate individuals
+whose probation sentence was imposed by a judge from a particular judicial district, determined by the
+judicial district that encompasses the county in which the offense took place.</p>`;
+
+const paroleBoilerplate = `<p>This data may include some individuals on parole in North
```
Comment (terryttsai) [inline spotlight-client/src/contentApi/sources/us_id.ts:51]: Should North Dakota be replaced with Idaho?

File: spotlight-client/src/contentApi/sources/us_tn.ts [hunk 4]
```diff
@@ -0,0 +1,766 @@
...
+the “controlling” charge, meaning the sentence associated with this charge is currently the
+most restrictive sentence being served. For individuals currently incarcerated, the sentence
+associated with the controlling charge determines the judicial district in which the person is
+counted. Individuals on supervision are counted in each judicial district for which they
+are serving a sentence.</p>`;
+
+const prisonBoilerplate = `<p>This data concerns individuals in prison in Tennessee.
+It does not include individuals incarcerated in county jails.</p>`;
+
+const probationBoilerplate = `<p>This data may include some individuals on probation in North
```
Comment (terryttsai) [inline spotlight-client/src/contentApi/sources/us_tn.ts:41]: Should North Dakota be replaced with Tennessee?

File: spotlight-client/src/contentApi/sources/us_tn.ts [hunk 5]
```diff
@@ -0,0 +1,766 @@
...
+Dakota as part of the interstate compact program, in which Tennessee agrees to supervise
+individuals who were charged and sentenced in a different state, but have a compelling reason
+to serve their probation in Tennessee. Under the same program, individuals placed on probation
+from a Tennessee court may occasionally serve their probation in a different state.</p>`;
+
+const probationDistrictBoilerplate = `<p>The districts associated with this data indicate individuals
+whose probation sentence was imposed by a judge from a particular judicial district, determined by the
+judicial district that encompasses the county in which the offense took place.</p>`;
+
+const paroleBoilerplate = `<p>This data may include some individuals on parole in North
```
Comment (terryttsai) [inline spotlight-client/src/contentApi/sources/us_tn.ts:51]: Should North Dakota be replaced with Tennessee?

**Pull request merged**